### PR TITLE
testSkip configuration added #73

### PR DESCRIPTION
**Pull request by confile**
... enables to skip Test

rename the cycleFinder to skipCycleFinder and negate the logic

### CODE REVIEW

File: j2objc.gradle [hunk 1]
```diff
@@ -99,7 +99,7 @@ class J2objcPluginExtension {
 
     // CYCLEFINDER
     // TODO(bruno): consider enabling cycleFinder by default
-    boolean cycleFinder = false
+    boolean skipCycleFinder = true
```
Comment (brunobowden) [inline j2objc.gradle]: Thinking about it again and I think that `cycleFinderSkip` and `testSkip` would be more consistent with the other flags that already exist. Please change that and then I'll merge it.

File: j2objc.gradle [hunk 2]
```diff
@@ -184,6 +184,8 @@ class J2objcPluginExtension {
 
     // TEST
     // Flags copied verbatim to testrunner command
```
Comment (brunobowden) [inline j2objc.gradle]: The "Flags copied verbatim..." comment should be next to the testFlags line of code

**Comment by confile** [comment]
done

**Comment by advayDev1** [comment]
LGTM, if indent on L748 is fixed

**Comment by brunobowden** [comment]
@confile - fix up L748 and then I'll merge

**Comment by confile** [comment]
@brunobowden What is wrong with L748?

**Comment by brunobowden** [comment]
See @advayDev1 comment about the indentation being off. Line 748 needs to be indented another space. See image:

screen shot 2015-05-21 at 8 38 52 am

**Comment by confile** [comment]
@advayDev1 @brunobowden  Thanks for comments I could not see the missing indentation in the github online editor.

**Comment by brunobowden** [comment]
It's now fixed. I might have had a stale copy before but it certainly
needed the indentation fix.

**Pull request merged**